config/v1/types_console.go

```diff
@@ -20,11 +19,15 @@ type Console struct {
 }
 
 type ConsoleSpec struct {
-	// special console public url?
+	// The default will be 3 replicas
+	Replicas int32 `json:"count,omitempty"`
```

Comment:
Replicas will be dropped.  We can make a sensible enough decision about this w/o needing configuration.